knowledge-base/system-methodaccessexception.md

```diff
@@ -0,0 +1,46 @@
+---
+title: System.MethodAccessException - Attempt by method 'System.Web.Http.HttpConfiguration..ctor(System.Web.Http.HttpRouteCollection)' to access method 'System.Web.Http.HttpConfiguration.DefaultFormatters()' failed
```

Comment:
This is too long for a title. Leave the whole error message in the description only and change the title and page_title tags to something smaller and different. Like: Getting MethodAccessException when ...